### PR TITLE
chore: document debug command

### PR DESCRIPTION
**Pull request by DanielMSchmidt**

### CODE REVIEW

File: website/docs/cdktf/cli-reference/commands.mdx [hunk 1]
```diff
@@ -546,3 +546,48 @@ The JSONs structure is matching your construct structure, the name of each const
 ```
 
 Please be aware that sensitive outputs are only written to the file if the `--outputs-file-include-sensitive-outputs` option is used.
+
+## debug
+
+This command gets debug information about the current project and environment to help troubleshoot issues.
```
Comment (ansgarm) [inline website/docs/cdktf/cli-reference/commands.mdx:552]:
```suggestion
This command prints debug information about the current project and environment to help troubleshoot issues.
```

File: website/docs/cdktf/cli-reference/commands.mdx [hunk 2]
```diff
@@ -546,3 +546,48 @@ The JSONs structure is matching your construct structure, the name of each const
...
+                                  CDKTF_DISABLE_PLUGIN_CACHE_ENV.                                                                             [boolean] [default: false]
+      --log-level                 Which log level should be written. Only supported via setting the env CDKTF_LOG_LEVEL                                         [string]
+      --json                      If set, output will be in JSON format.                                                                      [boolean] [default: false]
+  -h, --help                      Show help                                                                                                                    [boolean]
+```
+
+**Examples**
+
+The debug information collected depends on the language used in the current project in which the command is run.
+In this case it is a Java project, therefore we collect information about java and maven and detect the installed cdktf / constructs version through maven.
```
Comment (ansgarm) [inline website/docs/cdktf/cli-reference/commands.mdx:577]:
```suggestion
In this case it is a Java project, therefore we collect information about Java and Maven and detect the installed cdktf / constructs version through Maven.
```

**Review by ansgarm — APPROVED** [review]
looks got, only two small nits

**Review by laurapacilio — APPROVED** [review]
Approved with the following feedback. Also! Please add this new command to the linked list at the top of the page. We list out all of the commands at the top and provide a link to the heading so users can easily jump there.

**Pull request merged**